src/components/Articles/SingleArticle/SingleArticle.jsx

```diff
@@ -21,6 +26,27 @@ const SingleArticle = () => {
       });
   }, []);
 
+  const addLike = (article, num) => {
+    const copyArticle = { ...article };
```

Comment:
These 2 functions (addLike and takeLike) are being used in your ArticleButton component, so can they be removed here as they're not doing anything?